acceptance/users_test.go

```diff
@@ -107,11 +108,72 @@ var _ = Describe("Users", func() {
 			Expect(resp.StatusCode).To(Equal(http.StatusUnauthorized))
 		})
 	})
+
 	When("user doesn't exist", func() {
 		Specify("the response should be 401", func() {
 			resp, err := env.Client().Do(request)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(resp.StatusCode).To(Equal(http.StatusUnauthorized))
 		})
 	})
+
+	Describe("a regular user", func() {
+		var user, password string
+
+		BeforeEach(func() {
+			user, password = env.CreateEpinioUser("user", []string{"workspace"})
+		})
+
+		AfterEach(func() {
+			env.DeleteEpinioUser(user)
+		})
+
+		Specify("can describe its namespace", func() {
+			uri := fmt.Sprintf("%s%s/namespaces/workspace", serverURL, v1.Root)
+			request, err := http.NewRequest("GET", uri, nil)
+			Expect(err).ToNot(HaveOccurred())
+
+			request.SetBasicAuth(user, password)
+			resp, err := env.Client().Do(request)
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(resp.StatusCode).To(Equal(http.StatusOK))
+		})
+
+		Specify("cannot describe another namespace", func() {
+			uri := fmt.Sprintf("%s%s/namespaces/another", serverURL, v1.Root)
+			request, err := http.NewRequest("GET", uri, nil)
+			Expect(err).ToNot(HaveOccurred())
+
+			request.SetBasicAuth(user, password)
+			resp, err := env.Client().Do(request)
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(resp.StatusCode).To(Equal(http.StatusUnauthorized))
+		})
+	})
```

Comment:
Given the issue above with `CreateEpinioUser` I recommend adding a test where the user is linked to two or more namespaces.

Reply:
Agree, let's do this.